tools/base/envoy_python.bzl

```diff
@@ -0,0 +1,83 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+
+def envoy_py_library(
+        name = None,
+        deps = [],
+        data = [],
+        visibility = ["//visibility:public"]):
+
+    package = ".".join(name.split(".")[:-1])
```

Comment:
Nit: factor out `name.split`

Reply:
i dont follow

Reply:
oh i see - only split once - i can but i dont see the benefit

Reply:
i did this - but for simple string split and joins like this - i generally pref not to introduce the extra `_parts` line